test/integration/h2_capture_fuzz.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package test.integration;
+
+enum H2TestFrame {
```

Comment:
This might be asking too much for this PR, but I'm inclined to make this more general/extensible than just the frames created in `http2_frame.h`. 

Is it possible to make `TestFrame` line up with general H/2 frame type? https://http2.github.io/http2-spec/#iana-frames

Possibly following @htuch suggestions to make a oneof, can we make it a oneof with message type `bytes` so we can use these bytes as fuzzed inputs to the frame creation methods? (I'm imagining extended `http2_frame.h` for general "addHeader" methods, and use the bytes for this)

Reply:
Or messages, instead of bytes.

Reply:
The first version was essentially `bytes`, but then the probability to fuzz a valid H2 frame was low. That's why @htuch suggested restricting it to valid frame.
I think that, as you suggested, using messages would allow us to restrict what's valid to fuzz, and later on we can allow more fuzzing space by changing a frame's sub-fields to bytes.

Reply:
Sorry, to clarify, I don't mean to use `bytes` as the frame, but `bytes` to use as random inputs to flip options / convert to header names to feed to utility functions.

Reply:
Interesting, haven't thought of that.
I think that the advantage of using messages over `bytes` is the ability to change specific inputs by the fuzzer, and to allow specific extensions (e.g., stream_index field).

Reply:
Agreed, 10,000% more human readable.

Reply:
Yeah, +1 on message, driving from a byte vector is pretty hard to understand when reading the corpus.